packages/documentation/copy/en/javascript/JSDoc Reference.md

```diff
@@ -240,6 +221,28 @@ export const userAccount = {
 var x = require("./accounts").userAccount;
 ```
 
+### `@import`
+
+`@import` can be used for type-only imports:
+
+```js twoslash
+// @filename: types.d.ts
+export type Pet = {
+  name: string,
+};
+// @filename: main.js
+// ---cut---
+/**
+ * @import {Pet} from "./types"
+ */
+
+/**
+ * @type {Pet}
+ */
+var myPet;
+myPet.name;
+```
+
```

Comment:
````suggestion
```

These tags don't actually import files at runtime, and the symbols they bring into scope can only be used within JSDoc comments for type-checking.

```js twoslash
// @filename: dog.js
export class Dog {
  woof() {
    console.log("Woof!");
  }
}

// @filename: main.js
/** @import { Dog } from "./dog.js";

const d = new Dog(); // error!
````

You'll need to add the appropriate expected error to that code block though.